lib/l4p/tcp_rxtx.c

```diff
@@ -495,9 +527,8 @@ tx_nxt_data(struct tle_tcp_stream *s, uint32_t tms)
 			n = tx_data_bulk(s, &sl_una, mi, 1);
 			tn += n;
 			// of we enqueued the segment go back to normal processing
-			if (n == 1) {
+			if (n == 1)
```

Comment:
Abandon hope all ye who enter here...haha.  Call me old fashioned, but brackets for an if block just seems more natural than not.

Reply:
Welcome to TLDK/DPDK styling. I agree, but they generally don't.